screens/Settings/SettingScreen.tsx

```diff
@@ -154,13 +154,11 @@ export const SettingScreen: React.FC<SettingProps & RequestRouteProps> = (
                 progress={controller.isResetInjiProps}
               />
             )}
-            {/*
+
             <ListItem
```

Comment:
Please attach a screenshot for this change.

Reply:
Attached in the comments of resp. Jira ticket